ansible_ai_connect/users/reports/generators.py

```diff
@@ -125,6 +125,7 @@ def generate(
                 "First name",
                 "Last name",
                 "Email",
+                "Organization name",
```

Comment:
I imagine having the "Organisation name" column next to "Org ID" may have made Marty's life easier.

Reply:
I was following the same order we've got for the other report.